tests/unit/helpers/utils.test.js

```diff
@@ -257,20 +257,29 @@ describe('getDaysRatioBetweenTwoDates', () => {
   it('Case 2. Sundays in range', () => {
     const start = new Date('2019/05/18');
     const end = new Date('2019/05/23');
-    const result = UtilsHelper.getDaysRatioBetweenTwoDates(start, end);
+    const result = UtilsHelper.getDaysRatioBetweenTwoDates(start, end, true);
 
     expect(result).toBeDefined();
     expect(result).toEqual({ holidays: 0, sundays: 1, businessDays: 5 });
   });
 
-  it('Case 3. Holidays in range', () => {
+  it('Case 3. Holidays in range and shouldPayHolidays', () => {
     const start = new Date('2022/04/17');
     const end = new Date('2022/04/19');
-    const result = UtilsHelper.getDaysRatioBetweenTwoDates(start, end);
+    const result = UtilsHelper.getDaysRatioBetweenTwoDates(start, end, true);
 
     expect(result).toBeDefined();
     expect(result).toEqual({ holidays: 1, sundays: 1, businessDays: 1 });
   });
+
+  it('Case 4. Holidays in range and shouldPayHolidays', () => {
```

Comment:
je ne suis pas sûr du titre de ce test, ce n'est pas plutôt not shouldpayholidays ?

Reply:
Si, tu as raison